projects/packages/waf/src/class-waf-rules-manager.php

```diff
@@ -39,10 +39,9 @@ class Waf_Rules_Manager {
 	const IP_LISTS_ENABLED_OPTION_NAME = 'jetpack_waf_ip_list';
 
 	// Rule Files
-	const RULES_ENTRYPOINT_FILE = '/rules/rules.php';
```

Comment:
This constant is still used in the Debug Helper plugin on line 226 in `class-waf-helper.php`.

Reply:
Good catch! Fixed with [5782afb](https://github.com/Automattic/jetpack/pull/38944/commits/5782afb930bb92664d9cee07d19ef376058d1a04).